src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/legacy/controller/dto/LegacyPeriodLength.kt

```diff
@@ -13,22 +13,24 @@ data class LegacyPeriodLength(
   val isLifeSentence: Boolean?,
   val sentenceTermCode: String,
   val periodLengthUuid: UUID,
+  val sentenceUUID: UUID, // The LegacyPeriodLength only gets set when associated to a sentence atm, so safe to specify this as mandatory
```

Comment:
Added this comment because period-lengths can also be assoicated with a CourtAppearance and then dont have a sentence